commands/new_function.go

```diff
@@ -136,3 +138,50 @@ func validTemplate(lang string) bool {
 
 	return found
 }
+
+func contains(s []string, e string) bool {
+	for _, a := range s {
+		if a == e {
+			return true
+		}
+	}
+	return false
+}
+
+func createORUpdateGitignore() {
```

Comment:
Would be better if this returned an `error` instead of printing out to `fmt`. Then we can print in one place.

Reply:
updated and pushed the changes.